lib/commons/aria/subclass-implicit-nodes.js

```diff
@@ -0,0 +1,29 @@
+/* global aria */
+
+/**
+ * Get a list of CSS selectors of subclass nodes that have an implicit role
+ * @method subclassImplicitNodes
+ * @memberof axe.commons.aria
+ * @instance
+ * @param {String} role The role to check
+ * @return {Mixed} Either an Array of CSS selectors or `null` if there are none
+ */
+aria.subclassImplicitNodes = function(role) {
```

Comment:
I'd propose we create a `getRoleInheritance` function, which returns the role + recursively return any subclass roles.